### ISSUE TITLE
The results are quite different

### ISSUE THREAD

**Issue by yangwenxin847** [GitEventhandler/H2GCN-PyTorch#1]
Hello, when I downloaded the code and executed the Cornell dataset, there was a big gap between it and that in the paper, about 20%. At that time, I didn't change any parameters, so I want to ask you whether I didn't set any parameters in the citation

**Comment by GitEventhandler** [comment]
Hi! Thank you for report, Cornell dataset is a quite small dataset so I suppose the effect comes from backend is more significant. Result on bigger dataset such as PubMed seems more stable. It may need to modify hyper parameters such as seed, learning rate, hidden dim size, k value, or weight decay to reach the performance of the original implementation.
